addons/website_slides_forum/views/slide_channel_views.xml

```diff
@@ -6,8 +6,9 @@
         <field name="inherit_id" ref="website_slides.view_slide_channel_form"/>
         <field name="arch" type="xml">
     		<xpath expr="//notebook" position="inside">
-                <page string="Forum">
+                <page string="Communication">
                     <group>
+                        <field name="allow_comment"/>
```

Comment:
Should be done in website slides if this field exists in that module (or already present ? to check)

Reply:
You right !